atomic_red_team/attack_api.rb

```diff
@@ -75,7 +75,7 @@ def techniques_by_tactic(only_platform: /.*/)
     techniques_by_tactic = Hash.new {|h, k| h[k] = []}
     techniques.each do |technique|
       next unless !technique['x_mitre_platforms'].nil?
-      next unless technique['x_mitre_platforms'].any? { |platform| platform.downcase =~ only_platform }
+      next unless technique['x_mitre_platforms'].any? { |platform| platform.downcase.sub(" ", "-") =~ only_platform }
```

Comment:
Att&ck JSON file has platform values "Azure AD", "Google Workspace", etc. Atomic Red Team uses "google-workspace", "azure-ad".  Hence converting spaces to dashes here.